hw1/02-palindrome.js

```diff
@@ -1,3 +1,27 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+function handleInput() {
+  const inputNum = Number(elem.value);
+
+  if (inputNum > 0) {
+    // reverse the input
+    let reverseInput = elem.value.split("");
+    reverseInput.reverse();
+    reverseInput = reverseInput.join("");
```

Comment:
Just by the way, you can condense all of these into one line, each method call after another.